gradle.properties

```diff
@@ -40,7 +40,7 @@ org.gradle.jvmargs=-Xms256m -Xmx2048m -XX:MaxPermSize=256m
 gradleFury.version=1.0.14
 
 
-android-plugin.version=2.2.0
+android-plugin.version=4.1.3
```

Comment:
I would complete remove the variables here. An probably upcoming dependabot job, will make this obsolete anyways

Reply:
I'd like to keep this change focused and small, so a future PR could remove the variable.